base/threadingconstructs.jl

```diff
@@ -145,6 +145,7 @@ This includes both mark threads and concurrent sweep threads.
 ngcthreads() = Int(unsafe_load(cglobal(:jl_n_gcthreads, Cint))) + 1
 
 function threading_run(fun, static)
+    ccall(:jl_enter_threaded_region, Cvoid, ())
```

Comment:
Is this now only for the static nesting detection? This seems like we should use a ScopedValue for it xD.

Or for now move the code into Julia.